src/components/ApplicationGroup/Details/EnvironmentOverview/EnvironmentOverview.tsx

```diff
@@ -540,6 +562,15 @@ export default function EnvironmentOverview({
                     showDefaultDrawer={showDefaultDrawer}
                 />
             )}
+            {location.search && location.search.includes(URL_SEARCH_PARAMS.BULK_RESTART_WORKLOAD) && (
```

Comment:
we can write it as location.search?.includes(URL_SEARCH_PARAMS.BULK_RESTART_WORKLOAD) &&